stellar-contract-env-common/src/env_val.rs

```diff
@@ -209,7 +209,7 @@ impl<E: Env> IntoEnvVal<E, RawVal> for i64 {
 }
 
 impl<E: Env> TryFrom<EnvVal<E, RawVal>> for u64 {
-    type Error = ConversionError;
+    type Error = ConversionError<EnvVal<E, RawVal>, u64>;
```

Comment:
In my suggestion below what I mean is make this as follows, to indicate that it is a ConversionError destined for a `u64`:
```suggestion
    type Error = ConversionError<u64>;
```